resources/definitions/strateo3d.def.json

```diff
@@ -0,0 +1,136 @@
+{
+	"version": 2,
```

Comment:
Your indentation is a bit off at some locations. It's not a big issue, but it helps to keep it readable.

Reply:
Weird, i have no prob with indentation on my pc. I use sublime-text for editing and my indentation settings are set to tabs. should i set it to space maybe?

Reply:
Yeah, we use 4 spaces for indentation.

Reply:
just fixed it... thanx